rpc/Serve.go

```diff
@@ -0,0 +1,65 @@
+package rpc
+
+import (
+	"context"
+	"fmt"
+	"net"
+	"strings"
+)
+
+// Serve serves a Cap'n Proto RPC to incoming connections
+// Serve exits with the listener error
+func Serve(lis net.Listener, options *Options) error {
+	if options == nil { //|| !options.BootstrapClient.IsValid() {
```

Comment:
Hmm, after further consideration, I think a check on the  bootstrap interface being valid is needed. The NewConn might support Nil options but Serve will not function as intended. Rather than let the user figure out why it doesn't work, an error could inform them a valid bootstrap interface is required.

Reply:
>The NewConn might support Nil options but Serve will not function as intended.

How do you mean?  Any errors encountered will be reported as errors when you call methods on the bootstrap capability. 

Also, I think it's not as simple as calling `c.IsValid()` since the capability must first resolve.  Unless I'm missing something, I think it's much better to stay "hands off".

Reply:
As discussed, changed API to pass the bootstrap client instead of options